configs/config.json.example

```diff
@@ -30,7 +30,7 @@
     "use_lucky_egg": false,
     "hatch_eggs": true,
     "longer_eggs_first": true,
-    "evolve_captured": false,
```

Comment:
`false` will do the same as `"none"`, but `true` will not do the same as `"all"`. Added a parse error to cover that situation, as requested.